frontend/index.php

```diff
@@ -6,18 +6,11 @@
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
 
-$json = file_get_contents("/var/www/alldeals/frontend/deals.json");
+$json = file_get_contents("https://gianklug.com/alldeals-beta/frontend/deals.json");
 $data = json_decode($json, true);
 
-
-$digitecNumberOfItems = array_values($data["digitec"]["apidata"])[1]["salesInformation"]["numberOfItems"];
-$digitecNumberOfItemsSold = array_values($data["digitec"]["apidata"])[1]["salesInformation"]["numberOfItemsSold"];
-
-$galaxusNumberOfItems = array_values($data["galaxus"]["apidata"])[1]["salesInformation"]["numberOfItems"];
-$galaxusNumberOfItemsSold= array_values($data["galaxus"]["apidata"])[1]["salesInformation"]["numberOfItemsSold"];
-// calculate percentage still available
-$digitecPercentage = ($digitecNumberOfItems - $digitecNumberOfItemsSold) / $digitecNumberOfItems * 100;
-$galaxusPercentage = ($galaxusNumberOfItems - $galaxusNumberOfItemsSold) / $galaxusNumberOfItems * 100;
+$json =  file_get_contents("brandcolors.json");
```

Comment:
Let's not overwrite variables and name them `$deals_json` and `$colors_json` or something like that.

Reply:
fixed in c53f417